lib/awesome_bot/check.rb

```diff
@@ -9,9 +9,17 @@ module AwesomeBot
 
   STATUS_OK = '✓'
   STATUS_OTHER = 'x'
-  STATUS_REDIRECT = '→'  
+  STATUS_REDIRECT = '→'
 
   class << self
+    def check_output(s, log)
```

Comment:
I would suggest calling this `log_status` since it doesn't really check output, but it does log the status. It's marginally more descriptive. Again, just personal preference. 

Either way, refactoring this into a separate method is good idea. Good catch. :+1: